Code/Editor/EditorFramework/Assets/Implementation/AssetCurator.cpp

```diff
@@ -941,7 +945,7 @@ ezStatus ezAssetCurator::ProcessAsset(ezAssetInfo* pAssetInfo, const ezPlatformP
     if (assetFlags.IsAnySet(ezAssetDocumentFlags::DisableTransform))
       return ezStatus(EZ_SUCCESS);
 
-    if (!bTriggeredManually && assetFlags.IsAnySet(ezAssetDocumentFlags::OnlyTransformManually))
+    if (!transformFlags.IsSet(ezTransformFlags::TriggeredManually) && assetFlags.IsAnySet(ezAssetDocumentFlags::OnlyTransformManually))
```

Comment:
if (!transformFlags.AnyIsSet(ezTransformFlags::TriggeredManually | ezTransformFlags::Force)

Reply:
otherwise force would not have an effect

Reply:
Force would work if the asset is not triggered manually or if you set both. Otherwise force would imply triggered manually, do we want that?
In practice not really an issue as we never use force without TriggeredManually